source/whereAny.js

```diff
@@ -0,0 +1,52 @@
+import _curry2 from './internal/_curry2';
+import _has from './internal/_has';
+import _isEmpty from './isEmpty';
```

Comment:
there is no internal for `isEmpty`

Reply:
@CrossEye  any comment on this? **is it okay to import from non-internal**?

Reply:
I'm sure we do so in other places.  We have the internals mostly for when the auto-currying adds nothing but could conceivably slow things down.

Reply:
can be resolved, `isEmpty` is no longer being used.